requirements/requirements.txt

```diff
@@ -18,8 +18,8 @@ walrus==0.9.3
 flask-compress >= 1.0.0
 appdirs >= 1.0.0
 mindsdb-sql ~= 0.15.0
-pydantic >= 2.0.3
```

Comment:
Hey @tmichaeldb just a quick check here, that this doesn't affect other integrations that are using Pydantic? Are there any other required changes when upgrading? The last time this broke few integrations with depricated params provided?